### PR TITLE
add card & counter

### PR DESCRIPTION
**Pull request by yacheng**
*Before* submitting a pull request, please make sure the following is done...

1. Fork the repo and create your branch from `master`.
2. If you've added code that should be tested, add tests!
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`npm test`).
5. Make sure your code lints (`npm run lint`) - we've done our best to make sure these rules match our internal linting guidelines.

### CODE REVIEW

File: packages/rax-card-2colbigimg/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "rax-counter",
```
Comment (yuanyan) [inline packages/rax-card-2colbigimg/package.json]: packages/rax-card-2colbigimg/package.json 与 rax-counter 对应不上

File: packages/rax-card-2colbigimg/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+[![npm](https://img.shields.io/npm/v/rax-card-2colbigimg.svg)](https://www.npmjs.com/package/rax-card-2colbigimg)
```
Comment (yuanyan) [inline packages/rax-card-2colbigimg/README.md]: rax-card-2colbigimg 这个包名优化下

File: packages/rax-card/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "rax-counter",
```
Comment (yuanyan) [inline packages/rax-card/package.json]: name不对

File: packages/rax-counter/package.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "rax-counter",
+  "version": "0.3.1",
+  "description": "counter component for Rax.",
```
Comment (yuanyan) [inline packages/rax-counter/package.json]: 大小写细节

File: packages/rax-counter/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+## Props
+
+| name      | type       | default  | describe   |
+| :------ | :------- | :--- | :--- |
+| value | Boolean |      | default value |
+| start | Number |      | start num |
+| end | Number |      | end number |
+| onComplete | Object |      | complete event |
+| countStyle | Object |      | count style |
```
Comment (yuanyan) [inline packages/rax-counter/README.md]: 为什么不是 style  ？

File: packages/rax-counter/README.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+| incrementContent | Component |      | content in increment button |
```
Comment (yuanyan) [inline packages/rax-counter/README.md]: 参数是Component，命名上 xxxContent 不是直观，Content偏字符串

File: packages/rax-card-2col/README.md [hunk 7]
```diff
@@ -0,0 +1 @@
+[![npm](https://img.shields.io/npm/v/rax-card-2colbigimg.svg)](https://www.npmjs.com/package/rax-card-2colbigimg)
```
Comment (yuanyan) [inline packages/rax-card-2col/README.md]: 2colbigimg 没有改

File: packages/rax-card-2col/src/index.js [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+    overflow: 'hidden'
+  },
+  lineWrap: {
+    flexDirection: 'row',
+    alignItems: 'flex-end',
+    paddingLeft: 18,
+    marginTop: 8
+  },
+  price: {
+    fontFamily: 'PingFangSC-Medium',
```
Comment (yuanyan) [inline packages/rax-card-2col/src/index.js]: 无线上有这个字体？

Comment (yacheng) [inline packages/rax-card-2col/src/index.js]: iOS 9.0 以后默认支持

**Pull request merged**